manager/assets/modext/widgets/element/modx.panel.template.js

```diff
@@ -419,6 +436,27 @@ Ext.extend(MODx.panel.Template,MODx.FormPanel,{
                     }
                 }
             }
+        }*/
+        var fields = [
+            'modx-template-static-file',
+            'modx-template-static-file-help',
+            //'modx-template-static-source',
```

Comment:
Adding new code where half of the array is commented out doesn't make sense :) Include only what's needed.